exercises/say/tests/say.rs

```diff
@@ -13,13 +13,18 @@ fn test_zero() {
 // If the below test is uncommented, it should not compile.
 //
 /*
-#[test]
-fn test_negative() {
-    assert_eq!(say::encode(-1), String::from("won't compile"));
-}
+
+ #[test]
+ #[ignore]
+ #[should_panic]
```

Comment:
I'm unsure `should_panic` makes sense, since who can say whether the test should panic if it doesn't even compile? I would not add such a line.

I kind of wish Rust would have a `#[should_not_compile]` or something, but that is something to bring up with the Rust project, Is uppose

Reply:
> I kind of wish Rust would have a `#[should_not_compile]`

That is https://github.com/laumann/compiletest-rs apparently. a project for another day (and probably not something we can make students download)